internal/boxcli/global.go

```diff
@@ -128,6 +138,23 @@ func addGlobalCmdFunc(cmd *cobra.Command, args []string) error {
 	return box.AddGlobal(args...)
 }
 
+func installGlobalCmdFunc(cmd *cobra.Command, args []string) error {
+	path, err := ensureGlobalConfig(cmd)
+	if err != nil {
+		return errors.WithStack(err)
+	}
+
+	box, err := devbox.Open(path, cmd.ErrOrStderr())
+	if err != nil {
+		return errors.WithStack(err)
+	}
+	_, err = box.PrintEnv(cmd.Context(), false /* run init hooks */)
```

Comment:
wait, should this be `true` ?

Reply:
I think `false` is okay, because we want `install` to download the nix packages to `/nix/store` and update the global-nix-profile.

init-hooks are necessary when we are _also_ setting up the environment (which we are not here)